pywbem/cim_http.py

```diff
@@ -231,19 +231,19 @@ def timer_expired(self):
 
 
 def parse_url(url, allow_defaults=True):
-    """Return a tuple of ``(host, port, ssl)`` from the URL specified in the
-    ``url`` parameter.
+    """Return a tuple (host, port, ssl) from the URL specified in the
```

Comment:
Why no ` around host, etc. here but there is below?

Reply:
I did not go through the whole file because none of its functions is part of the external API. However, I did now and made it consistent.

Regarding the quotes: That was simply not consistent. I added one back-quote back in.